### PR TITLE
added move command

### LINKED ISSUE #583: Enhancement - move tracks
Add a way to change the order of tracks using a /move command.

### PR DESCRIPTION
**Pull request by therithwikrayani**
Closes #583 

Added a /move command. Moves song from a from index to a to index.

- [x] I updated the changelog

### CODE REVIEW

File: src/services/player.ts [hunk 1]
```diff
@@ -554,4 +554,14 @@ export default class {
       resolve(returnedStream);
     });
   }
+
+  move(fromindex: number, toindex: number): void {
```
Comment (codetheweb) [inline src/services/player.ts:558]:
```suggestion
  move(from: number, to: number): void {
```

**Comment by codetheweb** [comment]
Looks good! Could you update the changelog too?

**Comment by therithwikrayani** [comment]
ah I'm also supposed to sign the revert didn't know that

**Comment by therithwikrayani** [comment]
Had to do a force push as I messed up signing the revert.
Next time, I should amend the previous commit, got it.
Extremely sorry for the mess, this is my first time really contributing on GitHub.

**Pull request merged**